lib/src/chart/line_chart/line_chart_data.dart

```diff
@@ -162,6 +164,9 @@ class BelowBarData {
   final Offset gradientFrom;
   final Offset gradientTo;
 
+
+  final List<VerticalLine> verticalLines;
```

Comment:
here we should just have a single FlLine that determines the style of line, then we can repeat it for all spot points, we don't need VerticalLine, because we know where we should draw them (below the spots)

Reply:
Dropped in favor of FlLine